bumble/host.py

```diff
@@ -797,7 +798,9 @@ def on_hci_pin_code_request_event(self, event):
         # For now, just refuse all requests
         # TODO: delegate the decision
         self.send_command_sync(
-            HCI_PIN_Code_Request_Negative_Reply_Command(bd_addr=event.bd_addr)
+            HCI_PIN_Code_Request_Reply_Command(bd_addr = event.bd_addr,
```

Comment:
this hard-coded response doesn't seem to match the intent of responding to these events from device.py as supported in the other commit. Is that the intent here?

Reply:
It should be emitted to device I guess

Reply:
Updated, can you have another look?